modules/integration_azure_firewall/conf/01-snat.yaml

```diff
@@ -0,0 +1,20 @@
+## Example
+module: firewall
+name: "SNAT Port Utilization"
+
+transformation: false
```

Comment:
except if you really want to make this impossible to customize I would set it to true to provide a variable to user to override default behavior